pages/1.11/release-notes/1.11.7/index.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: layout.pug
+navigationTitle:  Release Notes for 1.11.7
+title: Release Notes for 1.11.7
+menuWeight: 5
+excerpt: Release notes for DC/OS 1.11.7
+---
+
+DC/OS 1.11.7 was released on November 1, 2018. 
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.11.7/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+
+DC/OS 1.11.7 includes the following components:
+- Apache Mesos 1.5.x [change log](https://github.com/apache/mesos/blob/5a7ad47e8fc1a14101e47a29eb8e7e2a20d959c5/CHANGELOG).
+- Marathon 1.6.549 [change log](https://github.com/mesosphere/marathon/tree/aabf74302).
+- Metronome 0.4.4 [change log](https://github.com/dcos/metronome/releases/tag/v0.4.4).
+
+
+# Issues Fixed in DC/OS 1.11.7
+
+## GUI
+- DCOS-20222 - Fix invalid environment variables. 
+- DCOS-42365 - Fix incorrect memory total in Pod instance.
+- DCOS_OSS-1961 - Support lifecycle of Pods in DC/OS UI.
+
+## Marathon
+- DCOS-43816 - Bump DC/OS OSS & EE to 1.11 for `marathon-1.6.564`.
+- DCOS_OSS-4193 - Marathon `systemd` unit requires zk-1.zk node to be available in order to launch Marathon service.
+
+## Mesos
+- COPS-3172/DCOS-40120/DCOS-42667 - Include MESOS-8545 and MESOS-9131 in Mesos bump.
+- DCOS-40309 - Bump Mesos to nightly 1.5.x 305c7f8.
+- DCOS-43236 - Fix UCR tests on strict mode clusters.
```

Comment:
This line of "fix UCR tests" needs to be removed.

Reply:
@yishan-lin could you provide the line summary for DCOS-43236? Is it "UCR tests on strict mode clusters."